lib/objectBuilder.js

```diff
@@ -431,13 +431,13 @@ module.exports = function () {
   function makeWizard() {
     var rec = _.assign(_createObject(), deviceInfo, {
       type: 'wizard',
-      recommended: REQUIRED,
+      recommended: OPTIONAL,
```

Comment:
These changes affect all the drivers, including those pushing up data through the platform. I did a double-check that all these fields are optional via jellyfish. Have you checked that they are all optional via platform?